sky/sdk/example/address_book/lib/main.dart

```diff
@@ -81,24 +100,25 @@ class AddressBookApp extends App {
   }
 
   Widget build() {
-
     ThemeData theme = new ThemeData(
       brightness: ThemeBrightness.light,
       primarySwatch: colors.Teal,
       accentColor: colors.PinkAccent[100]
     );
-
     return new Theme(
       data: theme,
-        child: new DefaultTextStyle(
-          style: typography.error, // if you see this, you've forgotten to correctly configure the text style!
-          child: new TaskDescription(
-            label: 'Address Book',
+      child: new DefaultTextStyle(
+        style: typography.error, // if you see this, you've forgotten to correctly configure the text style!
+        child: new TaskDescription(
+          label: 'Address Book',
+          child: new Focus(
+            defaultFocus: nameKey,
```

Comment:
Default feels a bit odd.  Why is it the job of the build function to change global state on first invocation?